src/util.ts

```diff
@@ -443,7 +449,7 @@ export function getCurrentGoPath(workspaceUri?: vscode.Uri): string {
 		}
 	}
 
-	const configGopath = config['gopath'] ? resolvePath(config['gopath'], currentRoot) : '';
+	const configGopath = config["gopath"] ? resolvePath(substituteEnv(config["gopath"]), currentRoot) : ""
```

Comment:
The introduction of double quotes and the missing semicolon in this line is causing linting errors.